src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -15,17 +16,17 @@ public class SampleDataUtil {
     public static Person[] getSamplePersons() {
         return new Person[] {
             new Person(new Name("Alex Yeoh"), new Phone("87438807"), new Email("alexyeoh@example.com"),
-                        new Address("Blk 30 Geylang Street 29, #06-40")),
+                        new Address("Blk 30 Geylang Street 29, #06-40"), new AdditionalNotes("")),
```

Comment:
I think you may want to add some artificial data into `additional notes` in order to make `SamplePersons` more similar to the real use scenario. Same applies for the latter `Persons`.